vscode-web/run.sh

```diff
@@ -72,14 +72,12 @@ for extension in "$${EXTENSIONLIST[@]}"; do
   output=$($VSCODE_WEB "$EXTENSION_ARG" --install-extension "$extension" --force)
   if [ $? -ne 0 ]; then
     echo "Failed to install extension: $extension: $output"
-    exit 1
   fi
 done
 
 if [ "${AUTO_INSTALL_EXTENSIONS}" = true ]; then
   if ! command -v jq > /dev/null; then
     echo "jq is required to install extensions from a workspace file."
```

Comment:
Oh wait we still need some kind of exit here.  Maybe put the block under an `else`?

Reply:
I guess it is ok to just give the message without exiting so that the script can continue.

Reply:
Let me know if I misunderstood though :laughing: 

```
if [ "${AUTO_INSTALL_EXTENSIONS}" = true ]; then
  if ! command -v jq > /dev/null; then
    echo "jq is required to install extensions from a workspace file."
  else
   WORKSPACE_DIR="$HOME"
   if [ -n "${FOLDER}" ]; then
     WORKSPACE_DIR="${FOLDER}"
   fi

   if [ -f "$WORKSPACE_DIR/.vscode/extensions.json" ]; then
     printf "🧩 Installing extensions from %s/.vscode/extensions.json...\n" "$WORKSPACE_DIR"
     extensions=$(jq -r '.recommendations[]' "$WORKSPACE_DIR"/.vscode/extensions.json)
     for extension in $extensions; do
       $VSCODE_WEB "$EXTENSION_ARG" --install-extension "$extension" --force
     done
   fi
  fi
fi
```